### PR TITLE
Added another example for a custom panel

### PR DESCRIPTION
**Pull request by SergeoLacruz**
Example with a button calling a function and explanations on how the code works.

### CODE REVIEW

File: docs/extend/plugins/panel.md [hunk 1]
```diff
@@ -55,3 +55,108 @@ note : see convention for template directory above.
 ## Example Implementation
 
 Refer to the `CustomPanelSample` example class in the `./plugin/samples/integration/` directory, for a fully worked example of how custom UI panels can be implemented.
+
+### An example with button 
+
+Lets have a look at another example. We like to have a new panel that contains a button.
```
Comment (SchrodingersGat) [inline docs/extend/plugins/panel.md:61]: Let's

File: docs/extend/plugins/panel.md [hunk 2]
```diff
@@ -55,3 +55,108 @@ note : see convention for template directory above.
 ## Example Implementation
 
 Refer to the `CustomPanelSample` example class in the `./plugin/samples/integration/` directory, for a fully worked example of how custom UI panels can be implemented.
+
+### An example with button 
+
+Lets have a look at another example. We like to have a new panel that contains a button. 
+Each time the button is clicked, a python function in our plugin shall be executed and
+do something usefull. The result will look like that:
```
Comment (SchrodingersGat) [inline docs/extend/plugins/panel.md:63]: useful

File: docs/extend/plugins/panel.md [hunk 3]
```diff
@@ -55,3 +55,108 @@ note : see convention for template directory above.
...
+</button>
+
+<br>
+{{ order.description }}
+{{ plugin.value }}
+{% endraw %}
+```
+
+We start with a bit of javascript. The function JGetCart just calls the url that has been defined in the python code above. 
+The url consists of a full path `plugin:plugin-name:url-namei`. The plugin-name is the SLUG that was defined in the plugin code.
```
Comment (SchrodingersGat) [inline docs/extend/plugins/panel.md:150]: Should be `plugin:plugin-name:url-name`, no?

**Review by SchrodingersGat — COMMENTED** [review]
@SergeoLacruz thanks for the extra example, a few simple spelling / text issues :)

**Comment by SergeoLacruz** [comment]
Thanks for checking.

**Comment by SchrodingersGat** [comment]
Thanks for the contribution @SergeoLacruz  :)

**Comment by eeintech** [comment]
@SergeoLacruz When adding new screenshots and copy-pasting the code, don't forget to update the `with id="xxx"` with something different for the `id` value each time as right now, all screenshot in `panel.md` are using `with id="panels"` and when you click on any of them it shows the first screenshot:

Click on this screenshot:
image

It shows:
image

**Comment by SchrodingersGat** [comment]
@eeintech good catch! @SergeoLacruz if you have time can you issue another quick PR to update?

**Pull request merged**